### PR TITLE
Check if action was not included in specs

### PR DESCRIPTION
**Pull request by dgusakov**

### CODE REVIEW

**Review by alexterent — APPROVED** [review]

**Review by AnatolyBobunov — DISMISSED** [review]

**Comment by dgusakov** [comment]
bender build

File: src/adcm_pytest_plugin/objects/actions.py [hunk 1]
```diff
@@ -23,6 +24,10 @@ def _get_bundle_id(bundle: Bundle):
     return f"{bundle.name}_{bundle.version.split('-')[0]}_{bundle.edition}"
 
 
+def _make_parent_name(prototype: Prototype):
+    return f"{prototype.name}_{prototype.display_name}"
```
Comment (skhomuti) [inline src/adcm_pytest_plugin/objects/actions.py:28]: probably better use `.` as a separator?

Comment (dgusakov) [reply src/adcm_pytest_plugin/objects/actions.py]: Done

**Review by Sealwing — APPROVED** [review]

**Review by skhomuti — APPROVED** [review]

**Review by AnatolyBobunov — APPROVED** [review]

**Comment by dgusakov** [comment]
bender build

**Pull request merged**